buildscripts/kokoro/unix.sh

```diff
@@ -70,6 +72,11 @@ fi
 LOCAL_MVN_TEMP=$(mktemp -d)
 # Note that this disables parallel=true from GRADLE_FLAGS
 if [[ -z "${ALL_ARTIFACTS:-}" ]]; then
+  if [[ $ARCH == "aarch_64" ]]; then
+    GRADLE_FLAGS+=" -x grpc-compiler:generateTestProto -x grpc-compiler:generateTestLiteProto"
```

Comment:
This is probably now unnecessary, since #6424 is now merged.

Reply:
Nevermind. This is because we can't run the compiled code since it is for a different architecture.